packages/optimizely-sdk/lib/shared_types.ts

```diff
@@ -80,29 +81,30 @@ export interface OptimizelyVariable {
 }
 
 /**
+ * The options object given to Optimizely.createInstance.
  * Entry level Config Entities
  */
-export interface SDKOptions {
-  // Datafile string
+export interface Config {
+  // TODO[OASIS-6649]: Don't use object type
+  // eslint-disable-next-line  @typescript-eslint/ban-types
   datafile?: string;
```

Comment:
We must maintain the same exact `Config` interface that was previously available.
```suggestion
  datafile?: object | string;
```